tern/analyze/oci/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.
```

Comment:
small nit: should be `2020` as the copyright year.

Reply:
I updated the copyright year.